client/post-editor/media-modal/index.jsx

```diff
@@ -210,7 +210,7 @@ module.exports = React.createClass( {
 		this.setView( ModalViews.IMAGE_EDITOR );
 	},
 
-	onImageEditorDone( error, blob, imageEditorProps ) {
+	onImageEditorDone( error, blob, imageEditorProps, media ) {
```

Comment:
I still maintain my position regarding this situation: `ID` shouldn't be passed to a component where its only one purpose is to edit an image.
Buuuttt ... otherwise, we could do something like ...

```es6
<ImageEditor
	siteId={ site && site.ID }
	id={ media.ID }
	media={ media }
	onDone={ imageEditionIsDone }
	onCancel={ this.onImageEditorCancel }
/>
```